app/flatpak-builtins-permission-reset.c

```diff
@@ -69,6 +69,7 @@ remove_for_app (XdpDbusPermissionStore *store,
       char *key;
       GVariant *value;
       GVariantBuilder builder;
```

Comment:
```suggestion
      g_auto(GVariantBuilder) builder = FLATPAK_VARIANT_BUILDER_INITIALIZER;
```
and get rid of the manual cleanup

Reply:
Done, thanks